### ISSUE TITLE
How to make the hidden states array output created by predict() the same?

### ISSUE THREAD

**Issue by Vic-TT** [hmmlearn/hmmlearn#380]
I tried to run a HMM model, some of the codes are as follow:
```
n = 3
model = GaussianHMM(n_components=n, covariance_type='full', n_iter=3000, random_state=0)
model.fit(fetures) 
res = model.predict(fetures)
```
so, I have run it for many times, and the res can be various,
sometimes the res can be [0,2,2,2,2,0,1]
sometimes the res can be [1,0,0,0,0,1,2]

it seems HMM is functioning well, because hidden states change at same positions, but the labels are different. But I want them to be the same for the future research, How can I do that?

After seeing a similar issue 'Training HMM with same data predicts different hidden states #271'，I tried to add 'np.random.seed(5)' in my code, but it seems not usefull, because the results are still different on two of my computers, when runing the same code. The new code are as follow:
‘’‘
np.random.seed(5)
model = GaussianHMM(n_components=n, covariance_type='full', n_iter=3000, random_state=0)
model.fit(fetures) 
res = model.predict(fetures)
’‘’

**Comment by Vic-TT** [comment]
I tried it again, 'np.random.seed(5)' can help.
